app/services/selection.js

```diff
@@ -163,7 +163,7 @@ export default Service.extend({
       }
 
       // all other transitions are done using attributes (tract can be inferred from block attributes, etc..)
-      this.explode(fromLevel, toLevel);
+      this.explodeGeo(fromLevel, toLevel);
```

Comment:
Are you sure? The function below is still named `explode`? Or is there a different function?

Reply:
Previously, some transitions were calculated using "explode" which used id lookups, and some were calculated using "explodeGeo" which uses spatial queries.  This has been changed so that all transitions use "explodeGeo".  I've cleaned up the unused code so this is better reflected.

Reply:
Ahh gotcha. Thanks!